mule-deployer/src/main/java/org/mule/tools/model/anypoint/CloudHubDeployment.java

```diff
@@ -41,6 +36,9 @@
   @Parameter
   protected Boolean persistentQueues = false;
 
+  @Parameter
+  protected Integer waitBeforeValidation = 0;
```

Comment:
Define a default value that will cause the minimun wait for the deployer to obtain the app status (e.g. 1 second). 
To avoid all users to define this property on their apps.

Reply:
Put it in 2secs (with only 1 it still fails)